[2024-05-14T08:08:38.889+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T08:08:38.899+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_jobs104.check_if_update manual__2024-05-14T08:08:31.231235+00:00 [queued]>
[2024-05-14T08:08:38.903+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_jobs104.check_if_update manual__2024-05-14T08:08:31.231235+00:00 [queued]>
[2024-05-14T08:08:38.903+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-14T08:08:38.908+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_if_update> on 2024-05-14 08:08:31.231235+00:00
[2024-05-14T08:08:38.911+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=59) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T08:08:38.912+0000] {standard_task_runner.py:63} INFO - Started process 61 to run task
[2024-05-14T08:08:38.913+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_jobs104', 'check_if_update', 'manual__2024-05-14T08:08:31.231235+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/.ipynb_checkpoints/data_pipeline_jobs104-checkpoint.py', '--cfg-path', '/tmp/tmpskkpyp67']
[2024-05-14T08:08:38.914+0000] {standard_task_runner.py:91} INFO - Job 69: Subtask check_if_update
[2024-05-14T08:08:38.935+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_jobs104.check_if_update manual__2024-05-14T08:08:31.231235+00:00 [running]> on host a2e728044965
[2024-05-14T08:08:38.977+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_jobs104' AIRFLOW_CTX_TASK_ID='check_if_update' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T08:08:31.231235+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T08:08:31.231235+00:00'
[2024-05-14T08:08:38.978+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T08:08:38.985+0000] {logging_mixin.py:188} INFO - check_if_update !!!
[2024-05-14T08:08:38.985+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T08:08:38.986+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T08:08:38.991+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_jobs104, task_id=check_if_update, run_id=manual__2024-05-14T08:08:31.231235+00:00, execution_date=20240514T080831, start_date=20240514T080838, end_date=20240514T080838
[2024-05-14T08:08:39.010+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T08:08:39.023+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-14T08:08:39.023+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
